types/2020-08-27/Quotes.d.ts

```diff
@@ -36,7 +36,7 @@ declare module 'stripe' {
        */
       application_fee_percent: number | null;
 
-      automatic_tax?: Quote.AutomaticTax;
+      automatic_tax: Quote.AutomaticTax;
```

Comment:
Should this change (and the tax_rate one) be documented in the changelog?

Reply:
Yes.